pypeit/scripts/multislit_flexure.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script enables the viewing of a raw FITS file
+"""
+from IPython.terminal.embed import embed
+import numpy as np
+import os
+
+from pypeit import par, msgs
+from pypeit.spectrographs.util import load_spectrograph
+from pypeit.par import pypeitpar
+from pypeit.core import flexure
+
+
+def read_flexfile(ifile):
+    """
+    Read a PypeIt .flex file, akin to a standard PypeIt file
+
+    The top is a config block that sets ParSet parameters
+
+    Args:
+        ifile (str):
+          Name of the flux file
+
+    Returns:
+        cfg_lines (list):
+          Config lines to modify ParSet values
+        spec1dfiles (list):
+          Contains spec1dfiles to be flexure corrected
+
+    """
+    # Read in the pypeit reduction file
+    msgs.info('Loading the flexure file')
+    lines = par.util._read_pypeit_file_lines(ifile)
+    is_config = np.ones(len(lines), dtype=bool)
+
+    # Parse the fluxing block
+    spec1dfiles = []
+    objids_in = []
+    s, e = par.util._find_pypeit_block(lines, 'flexure')
+    if s >= 0 and e < 0:
+        msgs.error("Missing 'flexure end' in {0}".format(ifile))
+    elif (s < 0) or (s == e):
+        msgs.error(
+            "Missing flexure read block in {0}. Check the input format for the .flex file".format(ifile))
+    else:
+        for ctr, line in enumerate(lines[s:e]):
+            prs = line.split(' ')
+            spec1dfiles.append(prs[0])
+            if len(prs) > 1:
+                msgs.error('Invalid format for .flex file.' + msgs.newline() +
+                           'You must have specify only spec1dfiles in the block ')
```

Comment:
"You must specify only spec1dfiles in the block"

Reply:
corrected